src/domain/profile/state/ProfileReducers.tsx

```diff
@@ -0,0 +1,28 @@
+import { createReducer } from 'redux-starter-kit';
+
+import { ProfileData } from '../types/ProfileTypes';
+import { PROFILE_ACTIONS } from '../constants/ProfileActionConstants';
+
+// TODO: Improve the structure of the data.
+export const defaultProfileValules: ProfileData = {
+  guardian: {
+    firstName: '',
+    lastName: '',
+    phoneNumber: '',
```

Comment:
We should save guardian data and children data separately instead of nest, even though i know they are nested